src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.inner_items.txt

```diff
@@ -1,9 +1,9 @@
     1|       |#![allow(unused_assignments, unused_variables, dead_code)]
     2|       |
     3|      1|fn main() {
-    4|       |    // Initialize test constants in a way that cannot be determined at compile time, to ensure
-    5|       |    // rustc and LLVM cannot optimize out statements (or coverage counters) downstream from
-    6|       |    // dependent conditions.
+    4|      1|    // Initialize test constants in a way that cannot be determined at compile time, to ensure
+    5|      1|    // rustc and LLVM cannot optimize out statements (or coverage counters) downstream from
+    6|      1|    // dependent conditions.
```

Comment:
Great question. I noticed this and compared the results with other test results. As it turns out, MOST of the tests already covered these identical leading comments, so from a consistency point of view, it's actually better.

As to why (academically), I suspect it's because I now check, when adding a span to the `refined_spans` to see if the new span is mergeable with the prior span, and if so, I extend the previous span to the end of the new span. This actually saves a counter. (Probably some small improvement in binary size and maybe execution time.)

I had to do this because the macro names (like `assert_eq!`) are now added preemptively (to ensure the macro call is counted, even if the macro expansion has branching logic). For macros that _DON'T_ have branches (no intermediate BCBs) the macro still gets a second `refined_span` for the span from the left paren to the right paren, but the second span is in the same BCB as the macro name. As a result, I saw one test that generated coverage results that looked something like:

```
    println!("some message");
    ^0        ^0
```

Merging spans, when possible, fixed this problem.